src/css/sections/header.css

```diff
@@ -1,33 +1,59 @@
 .header {
   padding: 20px 0;
 }
-.nav {
+
+.header .container {
   display: flex;
   justify-content: space-between;
 }
-.nav-logo {
-  display: inline-flex;
+
+.header-nav {
+  display: flex;
   align-items: center;
   justify-content: center;
+  margin-right: 16px;
+  flex-grow: 1;
+}
+.header-logo {
+  display: inline-flex;
+  max-width: 115px;
+  font-family: var(--font-primary);
+  font-size: 20px;
+  font-weight: 700;
+  line-height: 1.12;
+  letter-spacing: 0.02em;
+  margin-right: 76px;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.header-logo-accent {
+  color: rgb(var(--color-main));
+}
+
+.header-nav-list {
+  display: none;
 }
-.nav-list {
```

Comment:
nav.list should not contain @media query / example 

@media screen and (min-width: 768px) {
  .container {
    max-width: 768px;
  }
}